spec/factories/people.rb

```diff
@@ -5,6 +5,7 @@
     first_name { "Michael" }
     title { "Dr." }
     status { "Chair Representative" }
+    informations { [ Information.create(key: 'telegram', value: '@perscheid') ]}
```

Comment:
Why did you add this line of code?

Reply:
We have added tests to check if there are bagdes when searching for people. Before that, there was no information about a person and, accordingly, no badges.